aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Use ASP.NET Core APIs in a class library
+author: scottaddie
+description: Learn how to use ASP.NET Core APIs in a class library.
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 12/03/2019
+no-loc: [Blazor]
+uid: fundamentals/target-aspnetcore
+---
+# Use ASP.NET Core APIs in a class library
+
+By [Scott Addie](https://github.com/scottaddie)
+
+This document provides guidance for using ASP.NET Core APIs in a class library. For all other library guidance, see [Open-source library guidance](/dotnet/standard/library-guidance/).
+
+## Use the ASP.NET Core shared framework
+
+With the release of .NET Core 3.0, many ASP.NET Core packages are no longer published to NuGet. Instead, the packages are included in the `Microsoft.AspNetCore.App` shared framework, which is installed with the .NET Core SDK. The shared framework was previously distributed as a NuGet package. For a list of packages no longer being published, see [Remove obsolete package references](xref:migration/22-to-30#remove-obsolete-package-references).
```

Comment:
"With the release of .NET Core 3.0, many ASP.NET Core **assemblies** are no longer published to NuGet **as packages**. Instead, the **assemblies** are included in the `Microsoft.AspNetCore.App` shared framework, which is installed with the .NET Core SDK **and runtime installers**. "

Just remove the following line as it's more nuanced than that and it doesn't really add anything in this case: "The shared framework was previously distributed as a NuGet package."